prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -40,6 +46,8 @@
 # That is because at the closing time, the question will open on Realitio, and we don't want it to be resolved as unknown/invalid.
 # All replicated markets that close at N, needs to have closing time on Realition N + `EXTEND_CLOSING_TIME_DELTA`.
 EXTEND_CLOSING_TIME_DELTA = timedelta(days=6)
+# Banned categories are not allowed to be used for new markets on Omen.
+BANNED_CATEGORIES = ["Manifold Business Future"]
```

Comment:
I didn't understand from the thread - why is this category banned? Just because lots of questions in this category have titles that make an implicit assumption that 'manifold' if the subject?

Reply:
Yeah, and I thought that it's also not so interesting to copy markets about another PM platform.